lib/sql.ts

```diff
@@ -169,6 +162,25 @@ export class SQLDatabaseManager extends QueryProcessManager<DatabaseQuery, any>
 		};
 		if (!this.isParentProcess) this.setupDatabase();
 	}
+	private cacheStatement(source: string) {
+		source = source.trim();
```

Comment:
```suggestion
```
you already trim it in extractStatement()

Reply:
Yeah, but this trims here as well as a precaution in case you're doing `noPrepare`.

Reply:
`cacheStatement` isn't called when `noPrepare` is true, right? you just check if it's in the map directly

Reply:
Yeah, but that's still a safety check.

Reply:
`cacheStatement` is only called directly after trimming the text.

Reply:
it's trimmed in extract because in the this.state.statements.get call in exact, we want to ensure safety. It's not because of cacheStatement.